app/test_goldens/groups/src/pages/course/course_edit/design/src/dialog/select_design_dialog_test.dart

```diff
@@ -9,34 +9,121 @@
+import 'package:sharezone/sharezone_plus/subscription_service/subscription_service.dart';
+import 'package:sharezone_widgets/sharezone_widgets.dart';
 
+import 'select_design_dialog_test.mocks.dart';
+
+@GenerateNiceMocks([MockSpec<SubscriptionService>()])
 void main() {
   group('selectDesign', () {
-    testGoldens('display select design dialog as expected', (tester) async {
+    late MockSubscriptionService subscriptionService;
+
+    setUp(() {
+      subscriptionService = MockSubscriptionService();
+    });
+
+    Future<void> pumpSelectDesignDialog(
+      WidgetTester tester, {
+      ThemeData? theme,
+    }) async {
       await tester.pumpWidget(
-        MaterialApp(
-          home: Scaffold(
-            body: Center(
-              child: Builder(builder: (context) {
-                return ElevatedButton(
-                  // @visibleForTesting is not working for our `test_goldens`
-                  // folder. Therefore we have to ignore the warning.
-                  //
-                  // ignore: invalid_use_of_visible_for_testing_member
-                  onPressed: () => selectDesign(context, null),
-                  child: const Text("Select"),
-                );
-              }),
+        Provider<SubscriptionService>(
+          create: (_) => subscriptionService,
+          child: MaterialApp(
+            theme: theme,
+            home: Scaffold(
+              body: Center(
+                child: Builder(builder: (context) {
+                  return ElevatedButton(
+                    // @visibleForTesting is not working for our `test_goldens`
+                    // folder. Therefore we have to ignore the warning.
+                    //
+                    // ignore: invalid_use_of_visible_for_testing_member
+                    onPressed: () => selectDesign(context, null),
+                    child: const Text("Select"),
+                  );
+                }),
+              ),
             ),
           ),
         ),
       );
 
       await tester.tap(find.byType(ElevatedButton));
       await tester.pumpAndSettle();
+    }
+
+    group('with Sharezone Plus', () {
+      setUp(() {
+        when(subscriptionService
+                .hasFeatureUnlocked(SharezonePlusFeature.moreGroupColors))
+            .thenReturn(true);
+      });
+
+      testGoldens('displays select base color dialog as expected (light mode)',
+          (tester) async {
+        await pumpSelectDesignDialog(tester, theme: lightTheme);
+
+        await multiScreenGolden(tester, 'select_base_color_dialog_light');
+      });
+
+      testGoldens('displays select base color dialog as expected (dark mode)',
+          (tester) async {
+        await pumpSelectDesignDialog(tester, theme: darkTheme);
+
+        await multiScreenGolden(tester, 'select_base_color_dialog_dark');
+      });
+
+      testGoldens(
+          'displays select an accurate color dialog as expected (light mode)',
```

Comment:
```suggestion
          'displays select a color shade dialog as expected (light mode)',
```

I think this is better than "accurate color"